src/googmodule.ts

```diff
@@ -197,6 +197,37 @@ function rewriteModuleExportsAssignment(expr: ts.ExpressionStatement) {
       expr);
 }
 
+/**
+ * Convert a series of comma-separated expressions
+ *   x = foo, y(), z.bar();
+ * with statements
+ *   x = foo; y(); z.bar();
+ * This is for handling in particular the case where
+ *   exports.x = ..., exports.y = ...;
+ * which Closure rejects.
+ *
+ * @return An array of statements if it converted, or null otherwise.
+ */
+function rewriteCommaExpressions(expr: ts.ExpressionStatement): ts.Statement[]|null {
+  // Early exit if the outer statement isn't a comma statement.
+  if (!ts.isBinaryExpression(expr.expression)) return null;
+  if (expr.expression.operatorToken.kind !== ts.SyntaxKind.CommaToken) return null;
```

Comment:
if you wanted to be more precise, could you check for the `export` modifier (I think it's a modifier - or maybe a flag?) and/or check `ts.getOriginalNode`? I agree it should theoretically be harmless to convert like this, but we generally try not to muck with the user's original code if possible...

Reply:
Ah right, I had been thinking that this was working over the "es5 output" JS so the export flag is lost, but it's part of a transformation pipeline so it must still be there.  (In retrospect this is obvious because it manages to later jam the `exports.` in front.)  I think this is rare enough to not matter though.

Reply:
I started to write this, but it turns out that one of these forms ends up like:

```
_a = { a: 0, b: 0 }, exports.x = _a.a, exports.y = _a.b;
```

Where we still need to split to statements even though it has subexpressions that aren't export-related.

I could maybe try to special case it so we only split statements where one subexpression is an export, but I think I convinced myself again that this is a rare case.